src/fluree/db/peer/http_api.clj

```diff
@@ -112,9 +112,10 @@
 
 (defn decode-body
   [body type]
-  (case type
-    :json (let [^bytes body' (.bytes ^BytesInputStream body)]
-            (-> body' (String. "UTF-8") json/parse))))
+  (let [^bytes body' (.bytes ^BytesInputStream body)]
+    (case type
+      :string (String. body' "UTF-8")
+      :json   (-> body' (String. "UTF-8") json/parse))))
```

Comment:
If we want the string conversion in both cases it might make sense to do that up in the let form?

Reply:
✅